File: serverts/src/entities/UserInfo.ts
import { Column, Entity, JoinColumn, OneToOne, PrimaryGeneratedColumn} from "typeorm";
import { Users } from "./Users";

@Entity({name:'user_info'})
export class UserInfo{
    @PrimaryGeneratedColumn('uuid')
    id:string;

    @Column({length:280})
    description:string;

    @Column({length:5000, name:'perfil_image'})
    perfilImage:string;

    @OneToOne(()=>Users, user=>user.userInfo, {onDelete:'CASCADE'})
    @JoinColumn({name:'id_user'})
    user:Users
    
}

File: serverjava/src/main/java/com/proyecto/serverjava/entities/UserInfo.java
package com.proyecto.serverjava.entities;

import java.util.UUID;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import jakarta.persistence.CascadeType;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.OneToOne;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "user_info")
@Data
public class UserInfo {
    
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;

    @Column(length = 280)
    private String description;

    @Column(length = 5000, name="perfil_image")
    private String perfilImage;

    @OneToOne(cascade = CascadeType.PERSIST)
    @JoinColumn(name = "id_user")
    @JsonIgnoreProperties({"userInfo"})
    private Users user;
}
